demisto_sdk/commands/common/content/objects/pack_objects/pack.py

```diff
@@ -258,16 +259,28 @@ def upload(self, client: demisto_client):
         # turn off the sign check
         # upload
         # turn on the check
-        tools.update_server_configuration(client=client,
-                                          server_configuration={PACK_VERIFY_KEY: False},
-                                          error_msg='Can not turn off the pack verification')
+        logger.info('turn off the server verification for signed packs')
```

Comment:
```suggestion
        logger.info('Turning off the server verification for signed packs')
```

Please change all the logger outputs to use capital letters at the beginning of the sentence.